plugin/src/sidebar.ts

```diff
@@ -1,5 +1,6 @@
 import { DataCatalogItem } from "@web/extensions/sidebar/core/types";
 import { PostMessageProps, Project, PluginMessage } from "@web/extensions/sidebar/types";
+import _ from "lodash";
```

Comment:
Please don't import ALL of lodash. Just import the functions you need.
We have lodash's omit being imported directly from lodash/omit. Instead, please do `import {merge, omit} from "lodash";`

Reply:
done